nearlib/near.js

```diff
@@ -28,6 +28,25 @@ class Near {
         return bson.result;
     };
 
+    /**
+     * Schedules an asynchronous function call.
+     */
+    async scheduleFunctionCall(amount, sender, contractAccountId, methodName, args) {
+        if (!args) {
+            args = {};
+        }
+        const serializedArgs =  Array.from(BSON.serialize(args));
+        const response = await this.nearClient.submitTransaction('schedule_function_call', {
+            // TODO(#5): Need to make sure that big ints are supported later
```

Comment:
given the state of browser support for bigints (even Chrome doesn't support them in JSON yet), it's best to send them as strings basically.

BTW, not sure whether you really wanted to link this issue: https://github.com/nearprotocol/nearcore/issues/5

Reply:
Thanks, this was a copy paste from contract helper which lost relevance being moved to nearcore. Removed comment.